lib/razor/cli/command.rb

```diff
@@ -41,8 +41,10 @@ def extract_command
     body = {}
     until @segments.empty?
       argument = @segments.shift
-      if argument =~ /\A--([a-z-]+)(=(.+))?\Z/
-        # `--arg=value` or `--arg value`
+      if argument =~ /\A--([a-z-]{2,})(=(.+))?\Z/ or
+          argument =~ /\A-([a-z])(=(.+))?\Z/
+        # `--arg=value`/`--arg value`
+        # `-a=value`/`-a value`
```

Comment:
Most flags require an argument; it might be worth a future ticket to allow short flag combinations, but we can omit that for now I think.